R/augment.R

```diff
@@ -16,12 +16,28 @@ augment.mdl_vtl_df <- function(x, ...) {
 
 #' @export
 augment.mdl_vtl_ts <- function(x, ...) {
-  out <- response(x) |>
+  x_response <- response(x)
+  x_fitted <- x$fit$fitted
+  x_joined <- x_response |>
+    as_tibble() |>
+    merge(
+      x_fitted,
+      by = unname(c(vital_vars(x_response), index_var(x_response))),
```

Comment:
This should be `age_var()` rather than `vital_vars()`. It won't make any practical difference because Age is the only vital var that should be present at this point, but it makes the code more obvious that you are joining by age and index.